### PR TITLE
Update router terms

### PR DESCRIPTION
**Pull request by Meschreiber**
Updates uses of "Apollo Router" to "GraphOS Router" or "Apollo Router Core" where necessary

### CODE REVIEW

**Review by calvincestari — APPROVED** [review]
LGTM! Is there any specific guidance of when we should use "Apollo Router Core" vs. "GraphOS Router"?

**Comment by Meschreiber** [comment]
Thanks for the quick review! Yes, there's info [here](https://docs.google.com/document/d/1uflVIQ2jOPEhFXx3BHDUYEZk84xOGz0Ii036VAmM1Sc/edit), I believe Product leadership or PMM will be communicating about this change more broadly soon.

File: docs/source/file-uploads.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Enabling file uploads in Apollo Client for iOS
 Apollo iOS supports file uploads via the [GraphQL multipart request specification](https://github.com/jaydenseric/graphql-multipart-request-spec) with a few caveats:
 
 - Uploading files with GraphQL is most often suitable for proof-of-concept applications. In production, using purpose-built tools for file uploads may be preferable. Refer to this [blog post](https://www.apollographql.com/blog/backend/file-uploads/file-upload-best-practices/) for the advantages and disadvantages of multiple approaches.
-- The [Apollo Router](/router/) doesn't support `multipart/form-data` uploads.
+- The [Apollo Router Core](/router/) doesn't support `multipart/form-data` uploads.
```
Comment (shorgi) [inline docs/source/file-uploads.mdx:9]: Does this apply to Apollo Router Core and/or GraphOS Router?

**Review by AnthonyMDev — APPROVED** [review]
LGTM. Can we merge this, or are we awaiting approval from @shorgi?

**Comment by Meschreiber** [comment]
Apologies for the delay! Turns out there was some internal enablement about the terminology that still needs to happen. Will mark this as draft until that's been settled.

**Comment by Meschreiber** [comment]
Today's the day! @AnthonyMDev or @calvincestari could you please merge this whenever you get a chance? Thank you!

**Comment by BobaFetters** [comment]
Will merge this after I finish making a few tweaks to our PR merge workflow

**Pull request merged**